features/s3/postObject.feature

```diff
@@ -0,0 +1,7 @@
+@s3 @integ
+Feature: POST object uploads
+
+    Scenario: Upload and object via POST
+        Given I have an "s3" client
+        When I do a POST upload
```

Comment:
This step will need to be broken up into several steps that each do a single thing explicitly. Currently, the logic of how a POST upload is completed is contained in the context class instead of in this feature file.